patches/XSBench.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/openmp-offload/io.c b/openmp-offload/io.c
+index 5c981ee..c2fd0b7 100644
+--- a/openmp-offload/io.c
++++ b/openmp-offload/io.c
+@@ -275,8 +275,16 @@ Inputs read_CLI( int argc, char * argv[] )
+ 	{
+ 		char * arg = argv[i];
+ 
++		// This will be ignored
++		if( strcmp(arg, "-t") == 0 )
```

Comment:
Is an option on the number of threads correct? Shouldn't be 1 for correct program execution?